operator/duties/scheduler.go

```diff
@@ -158,6 +146,7 @@ func NewScheduler(opts *SchedulerOptions) *Scheduler {
 		ticker:   opts.SlotTickerProvider(),
 		reorg:    make(chan ReorgEvent),
 		waitCond: sync.NewCond(&sync.Mutex{}),
+		ctx:      opts.Ctx,
```

Comment:
I agree on skipping the genesis code instrumentation mainly because there is no reason to do that at this point. 
I removed `Scheduler` struct level context and instrumentation of `ExecuteGenesisDuties` method